pyproject.toml

```diff
@@ -1,10 +1,13 @@
 [tool.poetry]
 name = "up42-py"
-version = "0.32.0"
+version = "0.33.1"
 description = "Python SDK for UP42, the geospatial marketplace and developer platform."
 authors = ["UP42 GmbH"]
 license = "https://github.com/up42/up42-py/blob/master/LICENSE"
 readme = "README.md"
+packages = [
```

Comment:
@seedlit Please see this change, I think this solves the poetry error for the docs workflow and conda as well.